geoh5py/data/geometric_data.py

```diff
@@ -19,31 +19,22 @@
 
 import uuid
 
+from .data import Data
 from .primitive_type_enum import PrimitiveTypeEnum
 
 
-class GeometricDataConstants:
-    __X_TYPE_UID = uuid.UUID(
-        fields=(0xE9E6B408, 0x4109, 0x4E42, 0xB6, 0xA8, 0x685C37A802EE)
-    )
-    __Y_TYPE_UID = uuid.UUID(
-        fields=(0xF55B07BD, 0xD8A0, 0x4DFF, 0xBA, 0xE5, 0xC975D490D71C)
-    )
-    __Z_TYPE_UID = uuid.UUID(
-        fields=(0xDBAFB885, 0x1531, 0x410C, 0xB1, 0x8E, 0x6AC9A40B4466)
-    )
+class GeometricDataConstants(Data):
```

Comment:
But then, where this class is used.
I only so its import in data __init__

Reply:
The X, Y and Z are data of type `GeometricDataConstants`, of entity_type either GeometricDataX, Y or Z

Reply:
But now, GeometricDataX, Y, Z are not inherited from GeometricDataConstants...